doc/cjdns.md

```diff
@@ -112,5 +112,7 @@ There are several ways to see your CJDNS address in Bitcoin Core:
 To see which CJDNS peers your node is connected to, use `bitcoin-cli -netinfo 4`
 or the `getpeerinfo` RPC (i.e. `bitcoin-cli getpeerinfo`).
 
-To see which CJDNS addresses your node knows, use the `getnodeaddresses 0 cjdns`
-RPC.
+To fetch a number of CJDNS peers that your node knows, for example 7 addresses, run
+`bitcoin-cli getnodeaddresses 7 cjdns`. You may also run
+`bitcoin-cli getnodeaddresses 0 cjdns` to return all of them. These results, as well
```

Comment:
`These results, as well as those of -addrinfo, are filtered for quality and recency.`  - can you drop this. It's not clear why we need to mention an implemention detail of `-addrinfo` here.

Reply:
> You can use the getnodeaddresses RPC to fetch a number of onion peers known to your node; run bitcoin-cli help getnodeaddresses for details.

If anything, this seems better than reverting to the previous text.